404.html

```diff
@@ -0,0 +1,17 @@
+<html>
```

Comment:
```suggestion
<html lang="en">
```

Reply:
Oh fine I guess I should have valid xhtml haha